src/components/ListSchemas.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Space, Table } from "antd";
+import React, { useState } from "react";
+import { fetchAllSchemas } from "../services/chain/apis/extrinsic";
+import { SchemaDetails } from "../services/types";
+import { camelCaseToTitleCase } from "../common//util";
+import './styles.css';
+import Column from "antd/lib/table/Column";
+import CreateMessage from "./CreateMessage";
+import ListMessages from "./ListMessages";
+
+const ListSchemas = (): JSX.Element => {
+    const [listOfSchemas, setListOfSchemas] = React.useState<SchemaDetails[]>([]);
+    const [selectedSchema, setSelectedSchema] = useState<SchemaDetails>({key: 0, schema_id: "0", model_type: "", payload_location: "", model_structure: JSON.parse("{}")});
+    const [showCreateMessageComp, setShowCreateMessageComp] = useState(false);
+    const [showLlistMessageComp, setShowListMessageComp] = useState(false);
+
+    const listSchemas = async () => {
+        const schemas = await fetchAllSchemas();
+        setListOfSchemas(schemas);
+    }
+
+    return <div>
```

Comment:
Actually you don't need this at all, but `<>` over a `div`.
```suggestion
    return <>
```